### DISCUSSION TITLE
Account Restricted - How to access support?

### DISCUSSION THREAD

**Discussion by pspauster** [community/community#138413] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

My colleague had his account restricted and we can't figure out why. His account hosts a number of crucial daily actions that inform news gathering efforts for our local newsroom. But he can't access support because he can't access his account. How can we get in touch with account support to review the restriction?

**Comment by swapnilsparsh** [comment]
@pspauster Try to use different account and reach out to support and tell them your issue.

**Comment by davevad93** [comment]
Hi @pspauster ,

Tell your colleague to access GitHub with another account if he already has one or to create a new one and to contact support by opening a ticket [here](https://support.github.com/contact/account),  under "**Account or Organization**"  and "**What do you need help with?**" select `Other` and ask for clarification regarding his account restriction.
